src/renderer/preload-component.ts

```diff
@@ -68,7 +68,7 @@ const load = () => {
             component = NotificationComp;
             break;
         case components.notificationSettings:
-            document.title = 'Notification Settings - Symphony';
+            document.title = i18n.t('Notification Settings - Symphony', 'NotificationSettings')();
```

Comment:
I guess we will have to do it for all the components right

Reply:
We do it for all windows with a frame